index.html

```diff
@@ -16,41 +18,49 @@
 </script>
     <link rel="stylesheet" href="style1.css">
     <link rel="stylesheet" href="style2.css">
+   
     <title>Mugisha's Portfolio</title>
 </head>
 <body>
    <div class="hugo">
   <header class="header">
     <nav class="navbar">
+     
         <a href="#" class="nav-logo">UNIFY</a>
-        <ul class="nav-menu">
-            <li class="nav-item">
-                <a href="#" class="nav-link">Portfolio</a>
+      
+        <ul class="nav-menu ">
+          <li class="nav-item ">
+           <img src="images/close.png" class="close" alt="close-button">
+        </li>
```

Comment:
- [ ] Please note that the close button is not appearing on the page🤔🤷
![screenshot](https://user-images.githubusercontent.com/44624138/141001372-ac1bb483-ecd2-4d5e-8ec0-e9a679d2618a.png)